spec/features/post/post_index_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+RSpec.describe 'Post view', type: :feature do
+  include Devise::Test::IntegrationHelpers
+  describe 'Post index' do
+    let(:user) { User.create(name: 'denis', email: 'denis@example.com', password: 'password') }
+    describe 'display post information' do
```

Comment:
- [ ] Please add the following missing example to featured tests for Post index
1. Tests that asserts you can see the user's profile picture.
2. Tests that redirects to the post's show page when you click on a post.